wormchain/devnet/client/config/config.toml

```diff
@@ -249,7 +249,7 @@ send_rate = 5120000
 recv_rate = 5120000
 
 # Set true to enable the peer-exchange reactor
-pex = true
+pex = false
```

Comment:
why is pex disabled?

Reply:
If pex is enabled on non seed-nodes (in devnet, wormchain instances >= 1) they produce tons of errors as they try to find and connect to peers. We only need need pex on the seed node (`wormchain-0`), we don't need all nodes to be scanning the network for peers. Same reason/change as https://github.com/wormhole-foundation/wormhole/pull/2272.